File: Neuron.js
// Neuron class
class Neuron {
  constructor(bias){
    this.bias = bias || Math.random()
  }
  forward(input){
    var output = [];
    
    for(var i = 0;i < input.length;i++){
      output[i] = input[i] + this.bias;
    }
    
    return output;
  }
}

File: Neuron.py
import random

class Neuron:
  def __init__(self, bias):
    self.bias = bias or random.random()
  
  def forward(self, input):
    output = []
    
    for inputRow in input:
        n = 0
        for inputItem in inputRow:
            n += inputItem + self.bias

        output.append(n)
    
    return output
